### DISCUSSION TITLE
Unable to create the the first administrator remotely

### DISCUSSION THREAD

**Discussion by alocq** [keycloak/keycloak#10262] [Q&A] [answered]
### Describe the bug

Could you please help me out with creating the first administrator remotely, as I have only an SSH access to the server? 
Indeed, the procedure described here https://www.keycloak.org/docs/latest/server_admin/#creating-the-account-remotely is not working:

    export KEYCLOAK_ADMIN=admin
    export KEYCLOAK_ADMIN_PASSWORD=admin
    sudo bin/kc.sh start

### Version

keycloak-17.0.0

### Expected behavior

Created the first administrator as of the server gets started, upon the procedure above.

### Actual behavior

No administrator is created after the server has started

### How to Reproduce?

On Ubuntu 18.04

### Anything else?

Upon the procedure described above, the server has started without error, but no administrator was created.

**Comment by DirkLachowski** [comment, marked as answer]
If you export the environment variables and then start Keycloak with `sudo`, you get a new env. Either:
- start Keycloak as the user who has exported or
- do `sudo -E bin/kc.sh start` (see https://stackoverflow.com/a/8633575 )

**Reply by soundsessential** [reply]
Holy cow, thank you. Been battling at this for ages!

**Reply by deepaknaikhnvrm** [reply]
I was been struggling from yesterday. Since we are running keycloak in sudo the exported environment vars are not available but using -E helped.

**Comment by alocq** [comment]
Thank a lot DirkLachowski, you definitely RIGHT, that is it. By the way, it is working fine.
